### PR TITLE
Mention 'fix' for Chrome's broken PDF loading

### PR DESCRIPTION
**Pull request by lol768**
**Chromium team have disabled PDF partial loading entirely now due to issues it caused. This change obsoletes the previous issues and interaction with e.g. SameSite cookies as well as CORP.**

A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [x] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
  - https://bugs.chromium.org/p/chromium/issues/detail?id=1115149
- [x] Data: if you tested something, describe how you tested with details like browser and version
  - Did not test specifically
- [X] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
  - I assume it'll fail the build after I open this PR if I've screwed it up. If it does I'll clone locally and fix.
- [X] Link to related issues or pull requests, if any
  - #6026 #4088

### CODE REVIEW

**Comment by lol768** [comment]
The impression I get is Chrome will just download the entire blob now and won't try to issue e.g. Range requests.

File: http/headers/cross-origin-resource-policy.json [hunk 1]
```diff
@@ -9,14 +9,14 @@
               "version_added": "73",
               "notes": [
                 "Until version 75, downloads for files with this header would fail in Chrome. See <a href='https://crbug.com/952834'>bug 952834</a>.",
-                "From version 80, linearized PDFs served inline with this header fail to render properly. See <a href='https://crbug.com/1074261'>bug 1074261</a>."
+                "From version 80, linearized PDFs served inline with this header fail to render properly. See <a href='https://crbug.com/1074261'>bug 1074261</a>. Partial loading was disabled in v86, mitigating this bug."
```
Comment (ddbeck) [inline http/headers/cross-origin-resource-policy.json:12]: Can adjust this a bit to put the applicable versions up front?

```suggestion
                "From version 80 to 85, linearized PDFs served inline with this header fail to render properly. See <a href='https://crbug.com/1074261'>bug 1074261</a>. From version 86, partial PDF loading is disabled."
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
Thanks for this PR, @lol768. I have a small suggestion in-line. If the revised note makes sense, we could apply it to the others as well. Thank you again!

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Comment by lol768** [comment]
Thanks for reviewing :smiley:

**Pull request merged**